package.json

```diff
@@ -439,18 +439,20 @@
     }
   },
   "scripts": {
-    "clean": "rm -rf ./out ./server",
+    "clean": "rm -rf ./out",
+    "clean:server": "rm -rf ./server",
     "build": "webpack",
     "build:server": "git clone https://github.com/streetsidesoftware/vscode-spell-checker && cd vscode-spell-checker/packages/_server && yarn && yarn compile && mv ../client/server ../../../ && cd ../../../ && rm -rf vscode-spell-checker",
-    "build:all": "npm run clean && npm run build:server && npm run build",
-    "prepublishOnly": "npm run build:all"
+    "prepublishOnly": "npm-run-all clean clean:server build build:server",
```

Comment:
this should be `    "prepublishOnly": "npm-run-all clean clean:server build:server build"`

Reply:
fixed

Reply:
fixed